Doc/library/pathlib.rst

```diff
@@ -93,6 +93,128 @@ Opening a file::
+:meth:`read_text() <Path.read_text>`                                                                    Read the file as a string
+:meth:`read_bytes() <Path.read_bytes>`                                                                  Read the file as a bytes object
+:meth:`write_text() <Path.write_text>`                                                                  Write to the file as text
+:meth:`write_bytes() <Path.write_bytes>`                                                                Write to the file as bytes
+
+**Reading directories**
+--------------------------------------------------------------------------------------------------------------------------
+:meth:`iterdir() <Path.iterdir>`                                                                        Yield path objects of the directory contents
+:meth:`scandir() <Path.scandir>`                                                                        Iterator of :class:`os.DirEntry` objects corresponding to entries in the directory
+:meth:`glob() <Path.glob>`                                                                              Glob a pattern in the directory, yielding all matching files
+:meth:`rglob() <Path.rglob>`                                                                            Glob a pattern recursively
+:meth:`walk() <Path.walk>`                                                                              Generate the file names by walking the directory tree
+
+**Creating files and directories**
+--------------------------------------------------------------------------------------------------------------------------
+:meth:`touch() <Path.touch>`                                                                            Create a file at this given path
+:meth:`mkdir() <Path.mkdir>`                                                                            Create a new directory at this given path
+:meth:`symlink_to() <Path.symlink_to>`                                                                  Make this path a symbolic link to another file
+:meth:`hardlink_to() <Path.hardlink_to>`                                                                Make this path a hard link to another file
+
+**Copying, moving and deleting**
+--------------------------------------------------------------------------------------------------------------------------
+:meth:`copy() <Path.copy>`                                                                              Copy this file or directory tree to the given *target*
+:meth:`copy_into() <Path.copy_into>`                                                                    Copy this file or directory tree into the given *target_dir*
+:meth:`rename() <Path.rename>`                                                                          Rename this file or directory to the given *target*
```

Comment:
```suggestion
:meth:`rename() <Path.rename>`                                                                          Rename this file or directory to the given *target*. Raises an exception if the *target* already exists.
```

I think is clarification would be good.

Reply:
We don't need to add this second sentence.  This table is meant to help people find what they need, and then go read the full description.  There are many reasons why a method could raise an exception. This doesn't need to be mentioned in the summary.

Reply:
Should we not differentiate between rename/replace as they share the first sentence which may be confusing?

Reply:
Sorry, now I see what you mean.  Reading the full descriptions, it's even more subtle, and OS-specific.  I would go with:
```
rename: Rename this path to the given target if possible.
replace: Rename this path to the given target unconditionally.
```

Reply:
I don't mind which one you go with both mine and yours get the point across.